quantum/split_common/serial.c

```diff
@@ -14,6 +14,59 @@
 
 #ifndef USE_I2C
```

Comment:
This should probably be changed to something like
```suggestion
#if defined(USE_SERIAL) || (!defined(USE_I2C) && !defined(USE_SERIAL) )
```

Reply:
The USE_I2C,USE_SERIAL problem can be discussed by me or someone else in the next next phase. I will not change it now.

Reply:
I posted #4522. This is the solution to this problem I thought.